crates/aggchain-proof-builder/src/lib.rs

```diff
@@ -71,7 +71,8 @@ pub struct AggchainProofBuilderRequest {
     /// Aggchain proof partial prover inputs coming from the aggsender request.
     pub aggchain_proof_inputs: AggchainProofInputs,
 
-    /// Aggregation proof public values coming from the generated proof.
+    /// Aggregation proof's public values produced by the prover to verify the
+    /// proof.
```

Comment:
Nit: I find the formulation a little strange. How about one of the following?
```suggestion
    /// Aggregation proof's public values produced by the prover.
```
```suggestion
    /// Aggregation proof's public values produced by the prover. Used to verify
    /// the proof.
```